server/index.js

```diff
@@ -1,6 +1,6 @@
 const express = require('express');
 const path = require('path');
-const Review = require('../database/index.js');
+// const Review = require('../database/index.js');
```

Comment:
This comment might be able to be deleted if you dont think you weill ever use it again.